src/components/pages/HomePage/HomePage.jsx

```diff
@@ -276,136 +277,105 @@ const HomePage = () => (
                 <Icon link name='linkedin' className={styles.linkedIn} />
-              </a>
+              </a> */}
             </p>
           </Grid.Column>
           <Grid.Column width={2}>
-            <Image src={OctavianSalagean} size='medium' />
+            <Image src={MeabhMagee} size='medium' />
             <p>
               <span className={styles.p}>
-                Octavian V. Salagean
+                Méabh Magee
               </span>
               <br />
-              HackNotts Head of Finance
+              Social Media Secretary
               <br />
-              <a href='mailto:octavian@hacknotts.com' target='_blank' rel='noopener noreferrer'>
+              <a href='mailto:meabh.magee@hacksocnotts.co.uk' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='mail' color='green' />
               </a>
-              <a href='https://twitter.com/therebelliovs' target='_blank' rel='noopener noreferrer'>
+              {/* <a href='https://twitter.com' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='twitter' className={styles.twitter} />
-              </a>
-              <a href='https://linkedin.com/in/rebelliovs' target='_blank' rel='noopener noreferrer'>
+              </a> */}
+              {/* <a href='https://www.linkedin.com/in/' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='linkedin' className={styles.linkedIn} />
-              </a>
+              </a> */}
             </p>
           </Grid.Column>
           <Grid.Column width={2}>
-            <Image src={ViktoriaPopovici} size='medium' />
+            <Image src={JamesFrost} size='medium' />
             <p>
               <span className={styles.p}>
-                Viktoria Popovici
+                James Frost
               </span>
               <br />
-              HackNotts Head of Logistics
+              Marketing Secretary
               <br />
-              <a href='mailto:viktoria@hacknotts.com' target='_blank' rel='noopener noreferrer'>
+              <a href='mailto:james.frost@hacksocnotts.co.uk' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='mail' color='green' />
               </a>
-              <a href='https://twitter.com/viki_popovici' target='_blank' rel='noopener noreferrer'>
+              {/* <a href='https://twitter.com' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='twitter' className={styles.twitter} />
-              </a>
-              <a href='https://linkedin.com/in/viktoriapopovici' target='_blank' rel='noopener noreferrer'>
+              </a> */}
+              {/* <a href='https://www.linkedin.com/in/' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='linkedin' className={styles.linkedIn} />
-              </a>
+              </a> */}
             </p>
```

Comment:
```suggestion
            <a href="https://www.frost.cx">
              <Icon link name="globe" color="green">
            </a>
            </p>
```

Reply:
I'm guessing with the icon name, so that may need fixing.

Reply:
Just added this on #155